offset_request.go

```diff
@@ -1,19 +1,7 @@
 package sarama
 
-// OffsetTime is used in Offset Requests to ask for all messages before a certain time. Any positive int64
-// value will be interpreted as milliseconds, or use the special constants defined here.
-type OffsetTime int64
-
-const (
-	// LatestOffsets askes for the latest offsets.
-	LatestOffsets OffsetTime = -1
```

Comment:
I don't really see the difference. It's used in two different context, but in both contexts the mean the same thing: "I want the {oldest,newest} offset". I think having both `OffsetOldest` and `EarliestOffset` is way more confusing.